packages/blade/src/components/Collapsible/CollapsibleLink.tsx

```diff
@@ -0,0 +1,86 @@
+import type { ReactElement } from 'react';
+import { useCallback } from 'react';
+import styled from 'styled-components';
+import { useCollapsibleContext } from './CollapsibleContext';
+import type { CollapsibleProps } from './Collapsible';
+import type { IconComponent } from '~components/Icons';
+import { ChevronDownIcon } from '~components/Icons';
+import type { LinkProps } from '~components/Link';
+import { Link } from '~components/Link';
+import { BaseBox } from '~components/Box/BaseBox';
+import { castWebType, makeMotionTime } from '~utils';
+
+type CollapsibleLinkProps = Pick<
+  LinkProps,
+  'size' | 'isDisabled' | 'testID' | 'accessibilityLabel' | 'children'
+>;
+
+type StyledCollapsibleLinkIconProps = {
+  isExpanded: boolean;
+  direction: CollapsibleProps['direction'];
+};
+
+// TODO: refactor common transition properties
+const StyledCollapsibleLinkIcon = styled(BaseBox)<StyledCollapsibleLinkIconProps>((props) => {
+  const { isExpanded, direction, theme } = props;
+  let transformExpanded, transformCollapsed;
+  if (direction === 'bottom') {
+    transformExpanded = 'rotate(0.5turn)';
```

Comment:
TIL `turn` is a valid unit.

Reply:
Is browser support okay tho? maybe lets fallback to deg only.

Reply:
Don't see turn specific compatibility on rotate, I think it should be widely supported as deg.